TestFoundation/TestURLSession.swift

```diff
@@ -119,6 +120,56 @@ class TestURLSession : LoopbackServerTest {
         waitForExpectations(timeout: 12)
     }
     
+    func test_dataTaskWithHttpInputStream() {
+        func randomString(length: Int) -> String {
+            let letters = Array("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789")
+            let len = letters.count
+            
+            var randomString = ""
+            
+            for _ in 0 ..< length {
+                let rand = Int.random(in: 0..<len)
```

Comment:
Is using random data in tests a good idea? It makes tests runs non-reproduceable.

Reply:
I believe it not, I'll fix it.